static/app/views/explore/utils.tsx

```diff
@@ -0,0 +1,45 @@
+import * as qs from 'query-string';
+
+import type {PageFilters} from 'sentry/types/core';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import normalizeUrl from 'sentry/utils/url/normalizeUrl';
+import type {Mode} from 'sentry/views/explore/contexts/pageParamsContext/mode';
+import type {Visualize} from 'sentry/views/explore/contexts/pageParamsContext/visualizes';
+
+export function getExploreUrl({
+  orgSlug,
+  selection,
+  interval,
+  mode,
+  visualize,
+  query,
+  groupBy,
+  sort,
+}: {
+  interval: string;
+  orgSlug: string;
+  selection: PageFilters;
+  visualize: Omit<Visualize, 'label'>[];
+  groupBy?: string[];
+  mode?: Mode;
+  query?: string;
+  sort?: string;
+}) {
+  const {start, end, period: statsPeriod} = selection.datetime;
+  const {environments, projects} = selection;
+  const queryParams = {
+    dataset: DiscoverDatasets.SPANS_EAP_RPC,
+    project: projects,
+    environment: environments,
+    statsPeriod: statsPeriod,
+    start,
+    end,
+    interval,
+    mode,
+    query,
+    visualize: visualize.map(v => JSON.stringify(v)),
+    groupBy,
+    sort,
+  };
+  return normalizeUrl(`/organizations/${orgSlug}/traces/?${qs.stringify(queryParams)}`);
```

Comment:
It would be great if the /traces/ was a shared constants used in routes.tax in sidebar.txs

But that's outside of the scope of this PR imo